tests/backends/mysql/tests.py

```diff
@@ -38,8 +38,8 @@ def setUpClass(cls):
     @staticmethod
     def get_isolation_level(connection):
         with connection.cursor() as cursor:
-            cursor.execute("SELECT @@session.%s" % connection.transaction_isolation_variable)
-            return cursor.fetchone()[0]
+            cursor.execute("SHOW VARIABLES LIKE 't%_isolation'")
```

Comment:
Might be better to use `SHOW VARIABLES WHERE variable_name IN ('transaction_isolation', 'tx_isolation')`?

Reply:
good point, this avoids my laziness on the ticket :)